### ISSUE TITLE
BlockUI: CSS attribute not recognizing json 

### ISSUE THREAD

**Issue by cyanotyp** [primefaces-extensions/primefaces-extensions.github.com#642]
We currently migrate our application to PF 6.2 and PFE 6.2.10 on top.

The following json syntax (basically taken from your showcase) worked in PF5.2 with PFE 4.0.0
`<pe:blockui css="{backgroundColor: '#fff'}"> `  or `<pe:blockUI css="{top: '0px'}"`

with 6.2.10, without any change in our code, we now get a non-working blockui and the following Exception in Browser Console (Firefox 60 ESR) : 
"TypeError: 0 is read-only"

Basically I can work-around the same by using a css class and removing the attribute - but I think this is a bug.

**Comment by melloware** [comment]
Thanks for reporting. You are correct I just fixed it with this commit: https://github.com/primefaces-extensions/core/commit/0652adea5f85beb062c1e5c26b29ea2c0edefaf0

It will be in the upcoming PFE 7.0 release which will coincide with PF7.0 due to be released in the next couple of weeks.
